### ISSUE TITLE
Few things to look in *Deep Learning to extract visual features from posters* on Section 7

### ISSUE THREAD

**Issue by hskang9** [Spandan-Madan/DeepLearningProject#24]
1. You declared VGG model function and stored in variable 'model' and used variable 'model_viz' for training, which means you did not use VGG at all. You can check your model by typing 'print(model_viz.layers)'. If you struggle to fix this issue, I can help you with this section if you add me as an author.
2. It is important to show how well your model is trained. I would recommend plotting curves of loss and accuracy with history instance returned from 'model.fit()' function or a confusion matrix from predictions to show false positives and vice versa.
